src/ResourceManagement/HDInsight/HDInsight.Tests/HDInsight.Tests.csproj

```diff
@@ -5,6 +5,9 @@
     <AssemblyName>Microsoft.Azure.Management.HDInsight.Tests</AssemblyName>
     <Version>1.0.0</Version>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <SignAssembly>true</SignAssembly>
+    <DelaySign>true</DelaySign>
+    <AssemblyOriginatorKeyFile>..\Microsoft.Azure.Management.HDInsight\MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
```

Comment:
>..\Microsoft.Azure.Management.HDInsight\MSSharedLibKey.snk [](start = 31, length = 58)

Looks a little bit strange to me that key file is referenced from another project.

Suggest two options:
1. Move a copy MSSharedLibKey.snk under the same folder with `HDInsight.Tests.csproj`, then change this line as `<AssemblyOriginatorKeyFile>MSSharedLibKey.snk</AssemblyOriginatorKeyFile>`
2. Change this line to `<AssemblyOriginatorKeyFile>..\..\..\..\tools\MSSharedLibKey.snk</AssemblyOriginatorKeyFile>`

Reply:
modified based on 2 option